README.md

```diff
@@ -68,36 +68,86 @@ The frontend will display the contents of the results plugin for the AutoGov sta
 
 ### Installation into Backstage
 
+#### Update Configuration
+
+1. Add an autogov config section to your `app-config.yaml`
+
+```yaml
+autogov:
+  github:
+    resultsFile:
+      allowOverride: false
+      default: "results"
+    requireAnnotation: true
+    entityKinds:
+      - "component"
+    entityTypes:
+      - "website"
+      - "service"
+    maxReleasesResults: 5
+```
+
+1. Update GitHub integration if you have one
+
+```yaml
+github:
+  - host: github.com # or your github instance
+    apiBaseUrl: https://api.github.com
+    token: ${GITHUB_TOKEN}
+```
+
+#### Release Card install
+
 1. To install to the Autogov Plugins to the appropriate place, run the following two commands:
 
-   ```zsh
+```zsh
    # from root
    yarn --cwd packages/app add @liatrio/backstage-plugin-autogov-releases-card@^1.6.2
    yarn --cwd packages/backend add @liatrio/backstage-plugin-autogov-releases-backend@^1.6.2
-   ```
+```
 
 #### Autogov Plug-in Release Card Backend
 
 1. Update `packages/backend/src/index.ts` with:
 
+`diff`
+
 ```diff
 +  backend.add(
 +    import('@liatrio/backstage-plugin-autogov-releases-backend'),
 +  );
 backend.start();
 ```
 
+`copy paste`
+
+```.ts
+backend.add(
+  import('@liatrio/backstage-plugin-autogov-releases-backend'),
+);
+```
+
 #### Autogov Plug-in Release Card Frontend
 
 1. Update `packages/app/src/components/catalog/EntityPage.tsx` with:
 
-   ```diff
+`diff`
+
+```diff
    + import { AutogovReleasesCard } from '@liatrio/backstage-plugin-autogov-releases-card';
-   ```
+```
+
+`copy paste`
+
+```.tsx
+import { AutogovReleasesCard } from '@liatrio/backstage-plugin-autogov-releases-card';
+```
 
 1. Further down, find the following block of code:
 
-   ```diff
+`diff`
```

Comment:
If this is indented with tabs to be sub-level to the above `1.`, the numbering sequence will be automatically continued.